apps/info.c

```diff
@@ -10,11 +10,15 @@
 #include <openssl/crypto.h>
 #include "apps.h"
 #include "progs.h"
+#include <openssl/quic.h>
```

Comment:
does this need an ifdef?

Reply:
No, compiles fine in all combinations supported